apps/components/src/app/menu/menu.ts

```diff
@@ -0,0 +1,81 @@
+import { Component } from '@angular/core';
+import { NgpMenu, NgpMenuItem } from 'ng-primitives/menu';
+
+@Component({
+  selector: 'app-menu',
+  imports: [NgpMenu, NgpMenuItem],
+  template: `
+    <div ngpMenu>
+      <button ngpMenuItem>Item 1</button>
+      <button ngpMenuItem>Item 2</button>
+      <button ngpMenuItem>Item 3</button>
+    </div>
+  `,
+  styles: `
+    [ngpMenu] {
+      position: fixed;
+      display: flex;
+      flex-direction: column;
+      width: max-content;
+      background: var(--ngp-background);
+      border: 1px solid var(--ngp-border);
+      box-shadow: var(--ngp-shadow);
+      border-radius: 8px;
+      padding: 4px;
+      animation: menu-show 2s ease-out;
```

Comment:
Probably be good to shorten this duration:

```suggestion
      animation: menu-show 300ms ease-out;
```